mrchecker-framework-modules/mrchecker-database-module/pom.xml

```diff
@@ -59,7 +59,7 @@
     </scm>
 
     <properties>
-        <version>0.0.1-SNAPSHOT</version>
+
         
         <project.build.sourceEncoding>UTF8</project.build.sourceEncoding>
 	    <mrchecker-core-module.version>4.12.1.2</mrchecker-core-module.version>
```

Comment:
Still old version of core

Reply:
I meant we should have: <mrchecker-core-module.version>4.12.1.4-SNAPSHOT</mrchecker-core-module.version>